les/costtracker.go

```diff
@@ -264,9 +269,8 @@ func (ct *costTracker) gfLoop() {
 		for {
 			select {
 			case r := <-ct.reqInfoCh:
-				requestServedMeter.Mark(int64(r.servingTime))
```

Comment:
I used a Meter here (even though it is a time dimension value) because the exponential moving average of total served and estimated request times added up (regardless of the number of individual reqeusts) is a good indicator of how well the server cost estimation and self-regulation is performing. I think Timers can only measure the rate of individual events and the average/distribution of durations. Those could be interesting too, of course. Do you think it would make sense to use both Meters and Timers? (after all they are only used if we enable them)

Reply:
You are right, Meter is better to measure the exponential moving average and timer is better for value distribution. I personally think they are both valuable for system metrics. So I add both metric and timer.